daemon/cmd/kube_proxy_replacement.go

```diff
@@ -283,6 +283,16 @@ func finishKubeProxyReplacementInit(isKubeProxyReplacementStrict bool) {
 		return
 	}
 
+	if option.Config.EnableLoadBalancerSourceRangeCheck && !probe.HaveFullLPM() {
+		msg := "LoadBalancer source range check requires kernel 4.16 or newer."
```

Comment:
Please use the commandline argument string here to make it clearer to the user how they will change the configuration.